index.html

```diff
@@ -25,7 +25,7 @@ <h1>Luau Tickets</h1>
                     <input type="text" id="phonenumber" name="phonenumber" placeholder="Phone Number (xxx-xxx-xxxx)" autocomplete="off">
                     <input type="text" id="email" name="email" placeholder="Email" autocomplete="off">
                     <h3>Date of Birth</h3>
-                    <input type="date" id="birthday" maxlength="10" name="birthday" placeholder="(mm/dd/yyyy)" autocomplete="off">
+                    <input type="date" id="birthDate" maxlength="10" name="birthday" placeholder="(mm/dd/yyyy)" autocomplete="off">
```

Comment:
It looks like we have some inconsistent naming conventions. We have a hyphen on line 20, no spacing on line 25, and camel case on line 28. I went ahead and established a style convention for our HTML/CSS in the Wiki. We will stick with hyphens for our naming. JavaScript will still utilize camel cases.